buttons.js

```diff
@@ -24,6 +24,10 @@ const buttons = {
             .setLabel('Bedrock players')
             .setEmoji("<:bedrock:1089270506441687221>")
             .setStyle(ButtonStyle.Danger),
+        new ButtonBuilder()
+            .setCustomId('self_hosted')
```

Comment:
Please add an icon here, you can use .setEmoji, if you want a custom one please let us know and we can send u an emoji id :)

Reply:
alr gimme a sec